mungegithub/mungers/approval-handler.go

```diff
@@ -90,6 +90,7 @@ func (h *ApprovalHandler) Munge(obj *github.MungeObject) {
 	}
 
 	approverSet := sets.String{}
+	approverSet.Insert(*obj.Issue.User.Name)
```

Comment:
BCNTC (Better check null than crash, I just made that up). Applies specifically well when the data is being returned by a third party interface.